libminifi/src/sitetosite/SiteToSiteClient.cpp

```diff
@@ -467,12 +467,12 @@ int16_t SiteToSiteClient::send(std::string transactionID, DataPacket *packet, co
   }
 
   if (transaction->getState() != TRANSACTION_STARTED && transaction->getState() != DATA_EXCHANGED) {
-    logger_->log_debug("Site2Site transaction %s is not at started or exchanged state", transactionID.c_str());
```

Comment:
we should also make it a point to change these from transactionID.c_str() to tranactionID